react/tic-tac-toe/src/services/LocalStorageService.js

```diff
@@ -1,10 +1,18 @@
+function utoa(str) {
```

Comment:
Use a more descriptive name here, if you are enconding name it encode+Action

Reply:
👆